tools/ratbagd.py

```diff
@@ -755,6 +756,19 @@ def special(self, special):
         self._set_dbus_property("Mapping", "(uv)",
                                 (RatbagdButton.ActionType.SPECIAL, special))
 
+    @GObject.Property
+    def key(self):
+        type, key = self._mapping()
+        if type != RatbagdButton.ActionType.KEY:
+            return None
+        return key
```

Comment:
If I understand right, this will add a new property to the DBus object, which is equal to adding new API.
Sadly, because of this reason this PR will have to wait until we do API breaking.

Reply:
It's used in ratbagctl only. I've tried to remove it and Piper still works fine.